instagram-app/src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import "./PostContainer.css";
+import CommentSection from "../CommentSection/CommentSection";
+
+class PostContainer extends React.Component {
+  render() {
+    return (
+      <div className="post-container">
+        {this.props.filteredPosts.length === 0
+          ? this.props.data.map(post => {
```

Comment:
I really like how you handled the search and filter, and then this conditional render is excellent, really nice thinking. But, lines 12-38 should be refactored out into a singular post component. Notice how the code after line 43 is really similar but just created from a different array of information? That's an indicator that you should probably make a component out of it. Let me know if you need clarity on this.